metagraph/src/graph/alignment/aligner_extender_methods.cpp

```diff
@@ -19,6 +20,12 @@ constexpr score_t ninf = DBGAlignerConfig::ninf;
 constexpr size_t kPadding = 5;
 
 
+DefaultColumnExtender DefaultColumnExtender::make(const IDBGAligner &aligner,
+                                                  const DBGAlignerConfig &config,
```

Comment:
Ideally, it should only take `const IDBGAligner &aligner` and extract the config from it.

Reply:
done. This basically involved moving a bunch of stuff out of DBGAlignerConfig and passing them as arguments to `Extender::extend` instead